### ISSUE TITLE
Web Developer - Specializations: Machine Learning AI Healthtech Fintech

### ISSUE THREAD

**Issue by stevendelarwelle** [stevendelarwelle/remotejobs#6292]
Web Developer - Specializations: Machine Learning, AI, Healthtech, Fintech<br>
http://ift.tt/2AFzuT3<br>
<p><img src="http://ift.tt/2qDQadV" alt="Logo.gif?ixlib=rails 2.1"></p>
<p><strong>Headquarters:</strong> San Jose, CA<br>
<strong>URL:</strong> <a href="http://ift.tt/2BxH0RC">http://ift.tt/2BxH0RC</a></p>
<div><span>We are looking for web developers (both frontend and backend) to join a fast-growing global network of top-notch, hand-picked, freelancers in Machine Learning, AI, Healthtech &amp; Fintech. If you&rsquo;re a developer who not only has a broad technical skill-set but has the ability to understand a project&rsquo;s requirements and deliver them on time, come join GetIncredibles!</span></div>
<div><br></div>
<div>
<span>Benefits:</span><br>
</div>
<ul>
<li>
<span>Projects you&rsquo;ll love doing.</span><br>
</li>
<li>
<span>Work on projects that fit your schedule.</span><br>
</li>
<li>
<span>Forget finding and bidding for jobs online. GetIncredibles handles that hassle by finding jobs that suit your skill-set and negotiating hourly rates for you with clients.</span><br>
</li>
<li>
<span>Be a part of globally distributed teams consisting of highly-experienced Project Managers and vastly-skilled developers.</span><br>
<br>
</li>
</ul>
<div>
<span>Requirements:</span><br>
</div>
<ul>
<li>
<span>Must be fluent in English and agile-friendly</span><br>
</li>
<li>
<span>3+ years of experience building highly-scalable, high-traffic web apps in a professional software development environment.</span><br>
</li>
<li>
<span>2+ years of experience working remotely on various projects.</span><br>
</li>
<li>
<span>Track record of delivering &amp; launching high-traffic projects used by thousands of users.</span><br>
</li>
<li>
<span>Experience using source versioning software such as git, svn.</span><br>
</li>
<li>
<span>Experience with issue-tracking or bug-tracking software workflows in JIRA, Trac etc.</span><br>
</li>
<li><span>Have strong OOP concepts, and familiarity with RESTful APIs, MVC frameworks, scalable code.</span></li>
</ul>
<div><br></div>
<div>
<span>Good to-haves:</span><br>
</div>
<ul>
<li>
<span>Solid understanding of one of our core domains: Machine Learning, AI, Healthtech or Fintech.</span><br>
</li>
<li>
<span>One or more of the following:</span><br>
</li>
<li>
<span>Machine Learning algorithms like Deep Learning, Q-Learning etc. Having worked with TensorFlow would be a plus.</span><br>
</li>
<li>
<span>Knowledge of Neural Networks, classifying data and data mining. An advantage would be having worked on data mining.</span><br>
</li>
<li>
<span>Well-versed with HIPAA compliance, FDA approval processes etc. Direct experience building or maintaining Healthtech apps.</span><br>
</li>
<li><span>Experience building Fintech apps with emphasis on security and storing encrypted data would be a plus.</span></li>
</ul>
<div><br></div>
<div>
<span>What you can expect from us:</span><br>
</div>
<ul>
<li>
<span>After you sign-up on GetIncredibles.com, our recruitment team will contact you after reviewing your profile and work experience.</span><br>
</li>
<li>
<span>We will conduct a series of interviews with you at your convenience.</span><br>
</li>
<li><span>All applicants who qualify will be made an &lsquo;Incredible&rsquo; on our platform which means we will search for and match you to projects that you are passionate about and that fit your interests and skill-set.</span></li>
</ul>
<div><br></div>
<div><span>Please click the link below for the chance to join GetIncredibles, the fast-growing global network of top-notch, hand-picked freelancers in Machine Learning, AI, Healthtech and Fintech.</span></div>
<p><strong>To apply:</strong> Sign-up as an engineer at <a href="http://ift.tt/2BxH0RC">http://ift.tt/2BxH0RC</a></p>
<br>
<br>
via Remote Jobr Programming Jobs http://ift.tt/1WipnMk<br>
January 08, 2018 at 06:45AM
